contrib/colab/PyTorch_Bert_Multi_Core_TPU_Colab.ipynb

```diff
@@ -0,0 +1,865 @@
+            "Requirement already satisfied: s3transfer<0.4.0,>=0.3.0 in /usr/local/lib/python3.6/dist-packages (from boto3->transformers) (0.3.3)\n",
+            "Requirement already satisfied: jmespath<1.0.0,>=0.7.1 in /usr/local/lib/python3.6/dist-packages (from boto3->transformers) (0.9.4)\n",
+            "Requirement already satisfied: idna<2.9,>=2.5 in /usr/local/lib/python3.6/dist-packages (from requests->transformers) (2.8)\n",
+            "Requirement already satisfied: chardet<3.1.0,>=3.0.2 in /usr/local/lib/python3.6/dist-packages (from requests->transformers) (3.0.4)\n",
+            "Requirement already satisfied: certifi>=2017.4.17 in /usr/local/lib/python3.6/dist-packages (from requests->transformers) (2019.11.28)\n",
+            "Requirement already satisfied: urllib3<1.25,>=1.21.1 in /usr/local/lib/python3.6/dist-packages (from requests->transformers) (1.24.3)\n",
+            "Requirement already satisfied: six in /usr/local/lib/python3.6/dist-packages (from sacremoses->transformers) (1.12.0)\n",
+            "Requirement already satisfied: joblib in /usr/local/lib/python3.6/dist-packages (from sacremoses->transformers) (0.14.1)\n",
+            "Requirement already satisfied: click in /usr/local/lib/python3.6/dist-packages (from sacremoses->transformers) (7.0)\n",
+            "Requirement already satisfied: python-dateutil<3.0.0,>=2.1 in /usr/local/lib/python3.6/dist-packages (from botocore<1.15.0,>=1.14.15->boto3->transformers) (2.6.1)\n",
+            "Requirement already satisfied: docutils<0.16,>=0.10 in /usr/local/lib/python3.6/dist-packages (from botocore<1.15.0,>=1.14.15->boto3->transformers) (0.15.2)\n",
+            "Reading package lists... Done\n",
+            "Building dependency tree       \n",
+            "Reading state information... Done\n",
+            "libomp5 is already the newest version (5.0.1-1).\n",
+            "The following package was automatically installed and is no longer required:\n",
+            "  libnvidia-common-430\n",
+            "Use 'sudo apt autoremove' to remove it.\n",
+            "0 upgraded, 0 newly installed, 0 to remove and 25 not upgraded.\n"
+          ],
+          "name": "stdout"
+        }
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "ZZv23uP1pWrG",
+        "colab_type": "text"
+      },
+      "source": [
+        "# [IMP] Using Kaggle Google Quest Comp Dataset here For Demonstration purposes\n"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "ZK-gZNqoYcQk",
+        "colab_type": "code",
+        "outputId": "e037762b-2ce4-4579-dec3-ed1c44d6eea4",
+        "colab": {
+          "base_uri": "https://localhost:8080/",
+          "height": 247
+        }
+      },
+      "source": [
+        "# This Will Download the DataSet which is hosted on my Google Drive\n",
```

Comment:
We cannot have public Colabs depends on data hosted into personal Google Drives.

Reply:
Thanks a lot for all your comments! I can pretty much fix all the rest of them but not sure how to fix this one. Any thoughts?
Thanks Again!